sentry-android-core/src/main/res/layout/sentry_dialog_user_feedback.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/sentry_dialog_user_feedback_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:ignore="HardcodedText"
+    android:theme="?android:attr/dialogTheme"
+    android:padding="24dp">
+
+
+    <TextView
+        android:id="@+id/sentry_dialog_user_feedback_title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Report a Bug"
```

Comment:
You could use `"tools:text"` instead for some nice preview, as it all will be overwritten anyway right?